src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swr

```diff
@@ -2,8 +2,6 @@ use vs
 
 package name=Microsoft.VisualStudio.Editors
         version=15.0.0.0
-        vs.package.chip=neutral
```

Comment:
When the swixproj started, the setup authoring team wanted us to mention that details. Down the line, they have made those values as default values and have recommended to remove the repetition of default values. Today we get warnings while building the swixproj if the default values are explicitly mentioned!